agent/command/expansion_test.go

```diff
@@ -18,41 +18,224 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestExpansionsPlugin(t *testing.T) {
```

Comment:
I revamped this testing since it was like 6 years old and didn't cover a lot of the cases